src/jit/emit.cpp

```diff
@@ -3341,23 +3341,23 @@ void emitter::emitDispIGlist(bool verbose)
 void emitter::emitDispGCinfo()
 {
     printf("Emitter GC tracking info:");
-    printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
+    // printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
     dumpConvertedVarSet(emitComp, emitPrevGCrefVars);
```

Comment:
Yes, you are write, so is the right thing to do is just leave the 
`printf("\n  emitPrevGCrefVars ");` 
in the final change?